README.md

```diff
@@ -1,8 +1,28 @@
 # README #
 
 Colorectal cancer related scripts
+The wrapper will check if existing files in the path submitted contain modified files,
+and will modify according to original script if no modified files found (i.e. will not run
+the same files twice). The script will then compress the modified content and send a mail
+to signal that the files have been processed.
```

Comment:
Should also copy results to webstore and also backup selected information to HCP

Reply:
Yes, more "send-to"-stuff needs to be added!
And generally, thank you very much for all the input on this!
How do I do this in a good way now.. do I remove the pull-request and modify the code, and then create a new PR?